org.eclipse.pde.doc.user/whatsNew/pde_whatsnew.html

```diff
@@ -4,110 +4,17 @@
-    <td class="title">Support for names of product update repositories</td>
-    <td class="content">
-      The PDE Product Editor now supports in its <code>Updates</code> section to specify the <code>Name</code> of each update repository.
-      In the assembled product the names will be presented to a user in the preferences under <code>Available Software Sites</code>.
-      <p><img src="images/product_updates_names.png" alt="Product Updates with Name"/></p>
-    </td>
-  </tr>
-
-  <tr id="unnecessary-attributes-removal">
-    <!-- https://github.com/eclipse-pde/eclipse.pde/issues/730 -->
-    <!-- https://github.com/eclipse-pde/eclipse.pde/pull/770 -->
-    <!-- https://github.com/eclipse-equinox/p2/pull/378 -->
-    <!-- https://github.com/eclipse-pde/eclipse.pde/pull/882 -->
-    <td class="title">Removed support for unnecessary attributes in Features and Products</td>
-    <td class="content">
-      The <code>Feature</code> editor has its support for the following attributes of <code>plugin</code> elements removed:
-      <ul>
-        <li><code>download-size</code></li>
-        <li><code>install-size</code></li>
-        <li><code>unpack</code></li>
-        <li><code>fragment</code></li>
-      </ul>
-      These attributes are unused and without effect for a long time and unnecessarily increase the complexity of the editor and the size of a <code>feature.xml</code> file.
-      They are ignored when present in an existing Feature and removed by the editor upon the next modification through the editor.
-      <p>
-      The <code>Product Configuration</code> editor has its support for the following attribute of <code>plugin</code> elements removed:
-      <ul>
-        <li><code>fragment</code></li>
-      </ul>
-      This attribute is unused and without effect for a long time and unnecessarily increase the size of a <code>.product</code> file.
-      They are ignored when present in an existing Product and removed by the editor upon the next modification through the editor.
-      </p>
-    </td>
-  </tr>
-
-  <!-- ******************** End of Editors ********************** -->
-
-  <!-- ******************** APITools ********************** -->
-  <!--tr>
-    <td id="APITools" class="section" colspan="2"><h2>API Tools</h2></td>
-  </tr-->
-  <!-- ******************** End of APITools ********************** -->
-  
-   <!-- ******************** PDE Compiler ********************** -->
-  <!--tr>
-    <td id="pde-compiler" class="section" colspan="2"><h2>PDE Compiler</h2></td>
-  </tr-->
-   <!-- ******************** End of PDE Compiler ********************** --> 
-  <!--tr><td colspan="2"/></tr-->
-</tbody>
-</table>
-<!-- ****************** END OF N&N TABLE ****************** -->
-
+  <h2>Plug-in Development Environment</h2>
+  <p>
+    The new and noteworthy updates for Eclipse 4.37 can be found
+    <a href="https://eclipse.dev/eclipse/news/4.37/pde.html" target="_blank">here</a>.
```

Comment:
I wonder if we should embedd the content here with an `iframe` as well?

Also maybe it would be good to use maven filtered resources to automatically have this up to date always, e.g. the "copyright year" is also out of date.